flink-runtime/src/main/java/org/apache/flink/runtime/blob/PermanentBlobCache.java

```diff
@@ -54,7 +54,7 @@
  * <p>If files for a job are not needed any more, they will enter a staged, i.e. deferred, cleanup.
  * Files may thus still be be accessible upon recovery and do not need to be re-downloaded.
  */
-public class PermanentBlobCache extends AbstractBlobCache implements PermanentBlobService {
+public class PermanentBlobCache extends AbstractBlobCache implements JobPermanentBlobService {
```

Comment:
missing overrides for register-/releaseJob.

Reply:
Yes, they are missing. I will add them.